rust/parquet/src/column/writer.rs

```diff
@@ -216,26 +278,26 @@ impl<T: DataType> ColumnWriterImpl<T> {
             def_levels_sink: vec![],
             rep_levels_sink: vec![],
             data_pages: VecDeque::new(),
+            min_page_value: None,
+            max_page_value: None,
+            num_page_nulls: 0,
+            page_distinct_count: None,
```

Comment:
Seems this is not used at all?

Reply:
These are used to track page level stats and write those stats when writing a page and to update the column level stats when writing the page.

Reply:
Yes. But it is not updated nor used at all. Can you double check?

Reply:
Are you specifically referring to 'page_distinct_count', or all 4 page level vars?

Reply:
All of them are used, page_distinct_count isn't being calculated in this PR though, probably a following PR

Reply:
They're used here:

flush_data_pages()
make_typed_statistics()
update_page_min_max()

Reply:
Yes I meant `page_distinct_count`. It is fine to do this in a follow upo.

Reply:
How do I merge? Do I need to ask for merge/write permissions to complete the PR?